graphql/resolvers.go

```diff
@@ -1994,14 +1994,14 @@ func (r *mutationResolver) ScheduleUndispatchedBaseTasks(ctx context.Context, pa
 	return scheduledTasks, nil
 }
 
-func (r *mutationResolver) RestartPatch(ctx context.Context, patchID string, abort bool, taskIds []string) (*string, error) {
+func (r *mutationResolver) RestartPatch(ctx context.Context, patchID string, abort bool, taskIds []*model.TaskToRestart) (*string, error) {
```

Comment:
Can we do this in a backwards compatible way. 
We should create a new resolver called RestartVersion

Reply:
good point. I'll do that. 